test/sw_mgmt_spec.rb

```diff
@@ -178,8 +178,8 @@
       expect(File).to receive(:exist?).with(File.join(root_dir, target_dir, "SCCcredentials"))
         .and_return(false)
 
-      expect(FileUtils).to receive(:cp).with(File.join(root_dir, target_dir, "NCCcredentials"),
-        File.join(target_dir, "SCCcredentials"))
+      expect(subject).to receive(:`).with("cp -a " + File.join(root_dir, target_dir,
+        "NCCcredentials") + " " + File.join(target_dir, "SCCcredentials"))
```

Comment:
Oh, found it: The file is copied from `/mnt/etc/...` (from the target) to `/etc` in the inst-sys. There it is found by SUSEConnect, later (in the `*_finish` script) it is copied to the target system.

That's the reason for the asymmetry in the target directory...